src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java

```diff
@@ -0,0 +1,4 @@
+package org.checkstyle.suppressionxpathfilter.abstractclassname;
+
+public abstract class SuppressionXpathRegressionAbstractClassNameOne { // warn
```

Comment:
> SuppressionXpathRegressionAbstractClassNameOne 

it is better to make some reasonable names if possible.
For example:
SuppressionXpathRegressionAbstractClassNameTop
SuppressionXpathRegressionAbstractClassNameInner
SuppressionXpathRegressionAbstractClassNameNoModifier

test methods should be named in same patterns:
testModifierOne => testNoModifier

Reply:
Sure, let me change them.

Reply:
@romani Correct me if I am wrong, but all input files should start with `Input`.
It looks like our test to catch this isn't looking at the IT tier.

Reply:
lets proceed as we have now, after reply of @timurt , I will make a decision.